app/connectors/httpParsers/UserAnswersHttpParser.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors.httpParsers
+
+import connectors.errors.{ErrorResponse, FailedToDeleteUserAnswers, FailedToParseJson, FailedToStoreUserAnswers, UnexpectedErrorResponse}
+import models.UserAnswersModel
+import org.apache.pekko.Done
+import play.api.Logging
+import play.api.http.Status._
+import play.api.libs.json.{JsError, JsSuccess}
+import uk.gov.hmrc.http.{HttpReads, HttpResponse}
+
+object UserAnswersHttpParser extends Logging {
```

Comment:
I'm not sure I understand the nomenclature, but this isn't parsing the HTTP response, this is business layer handling of a full parse HTTP response. A better name would probably be `UserAnswersHttpReads`